source/_components/logger.markdown

```diff
@@ -11,14 +11,16 @@ logo: home-assistant.png
 ha_category: "Utility"
 ---
 
-The logger component lets you define the level of logging activities in Home Assistant.
+The `logger` component lets you define the level of logging activities in Home Assistant.
 
-To enable the logger in your installation, add the following to your `configuration.yaml` file:
+To enable the `logger` component in your installation, add the following to your `configuration.yaml` file:
```

Comment:
This sentence ends with a `:`, but so does the next? Aren't both saying almost the same thing?
IMHO those 2 need to be combined in 1 logical sentence...